app/routes/call/$slug.tsx

```diff
@@ -24,12 +25,17 @@ export default function Screen() {
   const params = useParams() as {slug: string}
   const episodes = useCallKentEpisodes()
   const episode = episodes.find(e => e.slug === params.slug)
+  const [theme] = useTheme()
+
   if (!episode) {
     return <div>Oh no... No episode found with this slug: {params.slug}</div>
   }
+
   return (
-    <div>
-      <pre>{JSON.stringify(episode, null, 2)}</pre>
-    </div>
+    <div
+      dangerouslySetInnerHTML={{
+        __html: theme === 'dark' ? episode.embedHtmlDark : episode.embedHtml,
+      }}
+    />
```

Comment:
Note that this doesn't work with fake data, as the iframe then doesn't load. Replace with the below for testing purpose:

`<iframe src='https://share.transistor.fm/e/1493e91f/dark' width='100%' height='180' frameborder='0' scrolling='no' seamless='true' style='width:100%; height:180px;'></iframe>`.

Reply:
We could update the mocks in `mocks/transistor.ts`